core/src/test/java/org/wso2/testgrid/core/command/RunTestPlanCommandTest.java

```diff
@@ -167,6 +177,12 @@ public void testExecute(String jobConfigFile) throws Exception {
         Assert.assertEquals(content, "Deploy server1\n", "my-deployment.txt file does not contain expected content.");
     }
 
+    private void copyWorkspaceArtifacts() throws IOException {
+        String resourceDir = "./src/test/resources/workspace";
+            FileUtils.copyDirectory(new File(resourceDir), new File(workspaceDir + "/workspace"));
+            logger.info("Copying necessary artifacts to workspace " + workspaceDir + "/workspace");
```

Comment:
Formatting issue. We need to think about File system differences in Windows and Linux.